interview-questions.md

```diff
@@ -7,52 +7,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props are objects that passed between classes.
 
-  Researched answer:
+  Researched answer: Props are arguments that are passed into React components.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event is any kind of instance where the user does something to the graphical user interface.
 
-  Researched answer:
+  Researched answer: A DOM event is any interaction with a user. These interactions can be when a user uses a keyboard to type or uses a mouse to click.
 
 
 
 3. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: Object-oriented programming is when software design is based around objects. Object-oriented programming is different from functional programming because they are different methods of programming.
 
-  Researched answer:
+  Researched answer: The difference between object-oriented programming and functional programming is that Object-oriented programming organizes software design around data or objects. Functional programming focuses on using functions to create clean and maintainable software.
 
 
 
 4. What is the difference between a Float and an Integer in Ruby?
 
-  Your answer:
+  Your answer: The difference between a float and an integer is that a float has a decimal and an integer does not.
 
-  Researched answer:
+  Researched answer: The difference between a float and an integer is that a float is a number that contains a decimal and an integer is any number without decimals.
 
 
 
 5. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: Implicit return will return an output even without a return statement.
 
-  Researched answer:
+  Researched answer: Implicit return will return the value of the last evaluated expression
```

Comment:
While your answers are correct in this section, one-line answers are the bare minimum and are not very interactive in an interview.  Your answer is a great time to relate to your personal experience with the concept and the researched answer is a great time to dig in deeper and discover other things about it.  When only responding with a single sentance, you take away the opportunity to connect with your interviewer and share your knowledge.